client-spark/spark-3/src/main/scala/org/apache/spark/shuffle/celeborn/CelebornShuffleReader.scala

```diff
@@ -121,7 +121,7 @@ class CelebornShuffleReader[K, C](
         }
         metricsCallback.incReadTime(
```

Comment:
IMO we need to count `openStream` time into read time because it belongs to the shuffle read phase